mm_graph.py

```diff
@@ -0,0 +1,252 @@
+import yfinance as yf
+import pandas as pd
+import plotly.graph_objs as go
+
+from models import Deal, symbol
+
+request_url = "https://www.nyse.com/api/regulatory/threshold-securities/" \
+              "filter?selectedDate={}&market=&filterToken=XRT&max=10&offset=0" \
+              "&pageNumber=1&sortOrder=up&sortType="
+
+data = yf.download(
+    # tickers list or string as well
+    tickers=symbol,
+
+    # use "period" instead of start/end
+    # valid periods: 1d,5d,1mo,3mo,6mo,1y,2y,5y,10y,ytd,max
+    # (optional, default is '1mo')
+    period="1d",
+
+    # fetch data by interval (including intraday if period < 60 days)
+    # valid intervals: 1m,2m,5m,15m,30m,60m,90m,1h,1d,5d,1wk,1mo,3mo
+    # (optional, default is '1d')
```

Comment:
Should this say `# (optional, default is '1m')`?

Reply:
This is from shamelessly stolen code from a tutorial, we should delete or condense it down

Reply:
There's still a lot of ground to cover. I'll handle this one as soon as I solve that problem with the .conf file we discussed yesterday.